.github/workflows/release.yaml

```diff
@@ -64,13 +67,6 @@ jobs:
           override: true
           default: true
 
-      - name: Downgrade cross
```

Comment:
This does not seem related; was this cleanup meant to be included in the PR?

Reply:
Well, it's kind of arbitrary, yeah. I started deleting this in { #129 #128 }. As it doesn't seem to have any effect I just deleted this one too without thinking about it.